std_msgs/Header header
float64 Theta1
float64 Theta2
float64 Theta3
float64 Theta4
float64 Theta5
float64 Theta6